IdentityCore/MSIDTestHostAppMacUITests/MSIDTestHostAppMacUITests.m

```diff
@@ -0,0 +1,48 @@
+//
+//  MSIDTestHostAppMacUITests.m
+//  MSIDTestHostAppMacUITests
+//
+//  Created by Ameya Patil on 9/29/21.
+//  Copyright © 2021 Microsoft. All rights reserved.
+//
+
+#import <XCTest/XCTest.h>
+
+@interface MSIDTestHostAppMacUITests : XCTestCase
+
+@end
+
+@implementation MSIDTestHostAppMacUITests
+
+- (void)setUp {
+    // Put setup code here. This method is called before the invocation of each test method in the class.
```

Comment:
nit: might be better to split this PR into two segments where you have another PR for this test app creation.

Reply:
The test cases in this PR needs to be run on Mac as well so I added this test app here.